src/util/FetchUtil.ts

```diff
@@ -10,6 +11,23 @@ import { parseContentType } from './HeaderUtil';
 
 const logger = getLoggerFor('FetchUtil');
 
+/**
+ * Add all RDF.Quads of a RDF.Stream to an array.
+ * @param quads- RDF.Stream of RDF.Quads
+ * @returns A Promise
+ */
+async function unrollQuads(quads: RDF.Stream<RDF.Quad>): Promise<RDF.Quad[]> {
+  return new Promise((resolve, reject): void => {
+    const arr: RDF.Quad[] = [];
+    quads
+      .on('data', (quad): void => {
+        arr.push(quad);
+      })
+      .on('error', (err): void => reject(err))
+      .on('end', (): void => resolve(arr));
+  });
+}
```

Comment:
You can use `arrayifyStream` for this (it's a library that CSS already depends on).

Reply:
Ok I will use that instead